packages/swirl-components/src/components/swirl-tree-view/swirl-tree-view.tsx

```diff
@@ -0,0 +1,255 @@
+import {
+  Component,
+  Element,
+  Event,
+  EventEmitter,
+  h,
+  Host,
+  Listen,
+  Method,
+  Prop,
+} from "@stencil/core";
+
+/**
+ * @slot slot - The tree view items
```

Comment:
**todo:** i think it's supposed to be `@slot tags - The tree view items`

Reply:
This component only has one unnamed slot for the items.

Reply:
ahh mixed them up sorry